Makefile

```diff
@@ -35,6 +35,14 @@ RELEASE_INFO := release-info.json
 
 MOCK_BUNDLE ?= false
 
+HYPERKIT_BUNDLENAME = crc_hyperkit_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
+HYPERV_BUNDLENAME = crc_hyperv_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
+LIBVIRT_BUNDLENAME = crc_libvirt_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
```

Comment:
Fwiw, there are a few occurrences of `crc_libvirt_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)` in the Makefile which could be replaced with `LIBVIRT_BUNDLENAME`